runner.py

```diff
@@ -274,6 +274,31 @@ def process_data(with_indels, dust, evalue, fastq_files, log, max_basecall_itera
     return reference_file
 
 
+def validate_input(output_dir, input_dir, reference_file):
+    if os.path.exists(output_dir):
+        list_dir = os.listdir(output_dir)
+        if len(list_dir) > 0:
+            if not ((len(list_dir) == 1) and (list_dir[0] == 'pbs_logs')):
+                raise Exception("output_dir must be path to a new or empty directory!")
+    if not os.path.isfile(reference_file):
+        raise Exception("Reference_file must exist!")
+    else:
+        file_name = os.fsdecode(reference_file)
+        if not file_name.endswith(".fasta"):
+            raise Exception("Reference_file must be fasta file!")
```

Comment:
what happens with the file doesn't end with .fasta? will it really fail?
and you are not checking that the file *is* a fasta file but only that it has the correct suffix so, you should specify that in the error message. otherwise a user can put in a correct file in the wrong format or vice versa and be confused.